src/components/themeModeIcon/themeModeIcon.tsx

```diff
@@ -0,0 +1,47 @@
+import styles from './themeModeIcon.module.scss';
```

Comment:
Component should be named uppercase - ThemeModeIcon

Reply:
Besides - this is not component which is only displaying icon..
more ThemeButton